pkgs/development/python-modules/scrapy/default.nix

```diff
@@ -55,6 +56,7 @@ buildPythonPackage rec {
     w3lib
     zope_interface
     protego
+    itemadapter
   ];
 
   patches = [
```

Comment:
patch is failing for me:
```
builder for '/nix/store/7sirkvkrsixvncgb01xmpjh4sq863k6p-python3.7-Scrapy-2.2.0.drv' failed with exit code 1; last 10 log lines:
  Sourcing setuptools-check-hook
  unpacking sources
  unpacking source archive /nix/store/sf0l6s4q10c5jzr41v6i75wy7ky0wpm9-Scrapy-2.2.0.tar.gz
  source root is Scrapy-2.2.0
  setting SOURCE_DATE_EPOCH to timestamp 1592998808 of file Scrapy-2.2.0/setup.cfg
  patching sources
  applying patch /nix/store/pajwq6m88bh2mr4kir28qqj76vpidn37-permissions-fix.patch
  patching file scrapy/commands/startproject.py
  Hunk #2 FAILED at 76.
  1 out of 2 hunks FAILED -- saving rejects to file scrapy/commands/startproject.py.rej
```

Reply:
Sorry about that, I meant to open this PR as draft.
With [Scrapy 2.2.1](https://github.com/scrapy/scrapy/releases/tag/2.2.1), this patch is no longer necessary:
>The `startproject` command no longer makes unintended changes to the permissions of files in the destination folder, such as removing execution permissions